application/server/server_test.go

```diff
@@ -799,6 +800,55 @@ func Test_textDocumentDidSaveHandler_shouldAcceptDocumentItemAndPublishDiagnosti
 	)
 }
 
+func createTemporaryDirectoryWithSnykFile(t *testing.T) (snykFilePath string, folderPath string) {
+	t.Helper()
+
+	temp := t.TempDir()
+	temp = filepath.Clean(temp)
+	temp, err := filepath.Abs(temp)
+	if err != nil {
+		t.Fatalf("couldn't get abs folder path of temp dir: %v", err)
+	}
+
+	snykFilePath = filepath.Join(temp, ".snyk")
+	yamlContent := `
+ignore:
+  SNYK-JS-QS-3153490:
+    - '*':
+        reason: Ignore me 30 days
+        expires: 2024-08-26T13:55:05.414Z
+        created: 2024-07-26T13:55:05.417Z
+patch: {}
+`
+	err = os.WriteFile(snykFilePath, []byte(yamlContent), 0600)
+	assert.Nil(t, err)
+	return snykFilePath, temp
+}
+
+func Test_textDocumentDidSaveHandler_shouldTriggerScanForDotSnykFile(t *testing.T) {
+	loc, jsonRPCRecorder := setupServer(t)
+	config.CurrentConfig().SetSnykCodeEnabled(false)
+	fakeAuthenticationProvider := di.AuthenticationService().Provider().(*snyk.FakeAuthenticationProvider)
+	fakeAuthenticationProvider.IsAuthenticated = true
+
+	_, err := loc.Client.Call(ctx, "initialize", nil)
+	if err != nil {
+		t.Fatalf("initialization failed: %v", err)
+	}
```

Comment:
same here - should use `assert.NoError(t, err)`

Reply:
This is the case when there is an error and all the other blocks are using `t.Fatal`. In this case I'm only appending the error message.